config/sequelize.js

```diff
@@ -32,8 +42,8 @@ fs.readdirSync(modelsDir)
 sequelize
     .sync()
     .then((err) => {
-        if (err) console.log('An error occured %j', err);
-        else console.log('Database synchronized');
+        if (err) console.log('An error occured %j', err); // eslint-disable-line no-console
```

Comment:
This may also be out of scope since it already existed.
This prints out "An error occured [Circular]" for me even though it appears to be working. I don't think `.sync()` returns an error on a resolved promise. See https://sequelize.readthedocs.io/en/latest/api/sequelize/#sync

We should probably put the error printing inside a `catch` rather than a `then`